src/utils/api.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios'
+import config from './config'
+import { getToken } from './auth'
+
+
+const token = getToken();
+
+const uri = config.BASE_URL;
+
+const headers = {
+  'Content-Type': 'application/json',
+  Authorization: `Token ${token}`,
+};
+
+const api = {
+  get(endpoint) {
+    return axios({
+      method: 'get',
+      url: `${uri}${endpoint}`,
+      data: {},
+      config: {
+        headers,
```

Comment:
Hey @charisschomba ,
I'm not sure these `headers` will work. I have used the same file `api.js` created by @imireallan and it doesn't work. Maybe you can refactor all the `http verbs` to include headers as follows:
```
post(endpoint, postData) {
    return axios({
      method: 'post',
      url: `${uri}${endpoint}`,
      data: postData,
      headers,
      },
    });
  },
```

Reply:
It works @SrMoffat